### ISSUE TITLE
Installer.sh and gitconfig

### ISSUE THREAD

**Issue by shudgston** [evansde77/cirrus#107]
* The `installer.sh` script follows a path that depends on `gitconfig` as a dependency, but `gitconfig` has been removed from requirements.txt.
* `bootstrap.py` is still importing `gitconfig`

Noticed this after a clean reinstall (removed ~/.cirrus).

```
[seanhudgston@duffman ~]$ bash installer.sh
Installation directory [/Users/seanhudgston/.cirrus]:
Installing cirrus in /Users/seanhudgston/.cirrus...
Cloning into 'cirrus'...
remote: Counting objects: 2447, done.
remote: Compressing objects: 100% (98/98), done.
remote: Total 2447 (delta 55), reused 0 (delta 0), pack-reused 2338
Receiving objects: 100% (2447/2447), 3.26 MiB | 4.68 MiB/s, done.
Resolving deltas: 100% (1347/1347), done.
Checking connectivity... done.
New python executable in /Users/seanhudgston/.cirrus/cirrus/venv/bin/python
Installing setuptools, pip, wheel...done.
Traceback (most recent call last):
  File "bootstrap.py", line 12, in <module>
    import gitconfig
ImportError: No module named gitconfig
```

Is installer2.sh meant to take installer.sh's place soon?

**Comment by evansde77** [comment]
Yeah, I updated the instructions to say use installer2. Its been a bit of a shit show switching over. Sorry :/

**Comment by shudgston** [comment]
Fixed in #108
